index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/src/favicon.svg" />
+    <link rel="icon" href="/src/logo.ico" type="image/x-icon">
```

Comment:
The updated logo has the modified logo, the rocket is a part of the logo. 
Can you please update the `.ico` to have the whole logo.

Reply:
Updated the .ico to have the rocket within the logo, but not the text for aesthetic reasons